testsuite/test-generate-hive.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+source test-generate-common.sh
+source test-common.sh
+source test-config.sh
+source test-generate-hadoop-helper.sh
+source test-generate-zookeeper-helper.sh
+
+__GenerateHiveStandardTests_StandardPerformanceEval() {
+    local hiveversion=$1
+    local hadoopversion=$2
+    local zookeeperversion=$3
+    local javaversion=$4
+
+    cp ../submission-scripts/script-${submissiontype}/magpie.${submissiontype}-hadoop-and-hive magpie.${submissiontype}-hadoop-and-hive-hadoop-${hadoopversion}-hive-${hiveversion}-zookeeper-${zookeeperversion}-hdfsoverlustre-sequential-thread-zookeeper-not-shared-zookeeper-networkfs-run-hiveperformanceeval
```

Comment:
i think naming this file with "sequential-thread" in the name is a cut and paste leftover?